docs/source/releases.rst

```diff
@@ -1,6 +1,76 @@
 Release History
 ###############
 
+
+v6.0.0 (2022-05-03)
+===================
+
+API Changes
+-----------
+- ``MultiDerivedSignal`` and ``MultiDerivedSignalRO`` calculation functions
+  (``calculate_on_get`` and ``calculate_on_put``) now take new signatures.
+  Calculation functions may be either methods on an ``ophyd.Device`` (with
+  ``self``) or standalone functions with the following signature:
+  .. code::
```

Comment:
I checked that this rendered properly when submitting that PR. I expect that it should be just fine here, too.